### PR TITLE
fix: a11y high contrast mode in storybook health factors

### PR DESCRIPTION
**Pull request by DarioR01**
Resolves WSTEAM1-179

**Overall change:**
Adds svg's preference style for high contrast mode in Windows.
This is to solve a bug found during the a11y swarm of WSTEAM1-179, where the title icon, won't change colour:
Screenshot 2023-03-06 at 12 37 32
In the above image, the background was change to green, and the text to white. However the title icon did not turn white, causing it to blend with the green background.
**Code changes:**

- Created a .styles file to better follow component standards.
- Refactor code to move the SVG colour on the parent span.

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: .storybook/DocsDecorator/HealthFactors/Icons/index.styles.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import pixelsToRem from '../../../../src/app/utilities/pixelsToRem';
+
+const styles = {
+  icon: (theme: Theme) =>
+    css({
+      display: 'block',
+      fill: 'currentcolor',
+      [theme.mq.HIGH_CONTRAST]: {
+        svg: {
+          fill: 'VisitedText',
```
Comment (greenc05) [inline .storybook/DocsDecorator/HealthFactors/Icons/index.styles.ts]: Why 'visited'?

Comment (DarioR01) [reply .storybook/DocsDecorator/HealthFactors/Icons/index.styles.ts]: I never used fill with attributes that are not colours, so I am unsure what is available to me. I will experiment with this today and find a working attribute.

**Comment by DarioR01** [comment]
Test changes:
High contrast mode edge (text: yellow):
image (4)
image (3)
image (2)


Firefox colour preference (text: blue):
Screenshot 2023-03-06 at 11 45 27
Screenshot 2023-03-06 at 11 45 20
Screenshot 2023-03-06 at 11 45 13

File: .storybook/DocsDecorator/HealthFactors/Icons/index.styles.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '../../../../src/app/utilities/pixelsToRem';
```
Comment (amoore108) [inline .storybook/DocsDecorator/HealthFactors/Icons/index.styles.ts]: Think `Theme` and `pixelsToRem` can be removed here as they aren't used.

**Review by HarveyPeachey — APPROVED** [review]

**Review by amoore108 — APPROVED** [review]

File: .storybook/DocsDecorator/HealthFactors/Icons/index.styles.ts (hunk 2 above)
Comment (DarioR01) [reply .storybook/DocsDecorator/HealthFactors/Icons/index.styles.ts]: Damn, that approval was fast 😂. Thank you for noticing this btw 👍

File: .storybook/DocsDecorator/HealthFactors/Icons/index.styles.ts (hunk 1 above)
Comment (DarioR01) [reply .storybook/DocsDecorator/HealthFactors/Icons/index.styles.ts]: FYI, I found this documentation about high contrast mode styling https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/
Might be good to know 🤷‍♂️

**Pull request merged**